clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -599,6 +599,10 @@ def LoadOp : CIR_Op<"load", [
     $addr `:` qualified(type($addr)) `,` type($result) attr-dict
   }];
 
+  let extraClassDeclaration = [{
+    void setAtomic(mlir::cir::MemOrder order);
```

Comment:
We already have a auto generated method for this setMemOrder, why do we need another?